common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1724,8 +1724,11 @@ def test_all_response_types(self, response_tag):
         self.assertEquals(descriptor.problem_types, {response_tag})
         self.assertEquals(descriptor.index_dictionary(), {
             'content_type': CapaDescriptor.INDEX_CONTENT_TYPE,
-            'display_name': name,
-            'problem_types': [response_tag]
+            'problem_types': [response_tag],
+            'content': {
+                'display_name': name,
+                'capa_content': ''
+            }
```

Comment:
Consumer of the `index_dictionary()` is our edx-search app that uses elasticsearch. Since we used  similar data scheme for the html xmodules, I don't see a need for changes to the edx-search.

I even tested this code on my devstack by indexing CAPA problems and searching for them from both dashboard and courseware pages, and both indexing and searching works like planned.